apps/etl/extraction/sources/base/handler.py

```diff
@@ -309,15 +309,22 @@ def _extraction_fetch_url(
         timeout: int = 30,
         file_extension: str = "json",
     ) -> bool:
-        if method == "get":
-            response = requests.get(url, params=params, headers=headers, timeout=timeout, proxies=self.proxies)
-        elif method == "post":
-            response = requests.post(url, headers=headers, data=data, timeout=timeout, proxies=self.proxies)
-        else:
-            typing.assert_never(method)
+        try:
+            if method == "get":
+                response = requests.get(url, params=params, headers=headers, timeout=timeout, proxies=self.proxies)
+            elif method == "post":
+                response = requests.post(url, headers=headers, data=data, timeout=timeout, proxies=self.proxies)
+            else:
+                typing.assert_never(method)
+        except requests.exceptions.Timeout as e:
+            logger.warning(
+                f"Request timed out, {self.extraction_object.attempt_no}",
```

Comment:
@sudan45 Either remove the attempt number or clearly mention it as Attempt number.

Reply:
@sudan45 We need to add time out for emdat aswell in `def _extraction_fetch_graphql`